src/components/formComponentTemplateEndpoints.tsx

```diff
@@ -906,6 +906,31 @@ export const formComponentTemplateEndpoints = (
         ),
         validate: validate.getPositiveIntError,
       };
+    case "xdrFormat":
+      return {
+        render: (templ: {
+          value: string | undefined;
+          error: string | undefined;
+          onChange: (val: any) => void;
+        }) => (
+          <Select
+            key={id}
+            id={id}
+            fieldSize="md"
+            label="XDR Format"
+            value={templ.value || ""}
+            onChange={templ.onChange}
+          >
+            <option id="base64" value="base64" selected>
```

Comment:
Nit: we don't need `selected` attribute here. It will be selected by default as the first option on the list.